### PR TITLE
run the sample codes added by `add_sample_code` in ops.py

### PR DESCRIPTION
**Pull request by wadefelix**
### PR types
Bug fixes

### PR changes
Others

### Describe

https://console.cloud.baidu-int.com/devops/icafe/issue/DLTP-23536/show

1 bugfix: the sample codes added py `add_sample_code` not run in Doc CI, all then are in the file : https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/layers/ops.py 
2 using `argparse` in `sampcd_processor.py` to handle the program parameters.
3 unittests `test_sampcd_processor.py` for `sampcd_processor.py`
~~4 add class `Flowers` into the wlist_temp_api, IT waste so many times to print the downloaded datasets.~~

see https://xly.bce.baidu.com/paddlepaddle/paddle/newipipe/detail/2585244/job/3605231 , I modified a op's docstring in https://github.com/wadefelix/Paddle/commit/f0dac636eb5fa8d5ee0072b69d8c82005b2a8a88 , triggered successfully.
see https://xly.bce.baidu.com/paddlepaddle/paddle/newipipe/detail/2587053/job/3608290 , empty the API_DEV.spec https://github.com/PaddlePaddle/Paddle/pull/31863/commits/d35fac7f39749f44399efa0061966dc1015198ef to run all the sample codes. YES, all the sample codes are extracted, but some tests failed for some reason, such as missing `enable_static()`.

### CODE REVIEW

File: tools/test_sampcd_processor.py [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+            os.path.exists("samplecode_temp/four_plus_four_example.py"))
+        self.assertFalse(
+            os.path.exists("samplecode_temp/three_plus_three_example.py"))
+
+
+# https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/layers/ops.py
+# why? unabled to use the ast module. emmmmm
+
+if __name__ == '__main__':
+    unittest.main()
```
Comment (lelelelelez) [inline tools/test_sampcd_processor.py:439]: 这个单测看看是不是需要加入到全量单测中？@XieYunshen

**Review by lelelelelez — APPROVED** [review]
LGTM

File: tools/test_sampcd_processor.py (hunk 1 above)
Comment (XieYunshen) [reply tools/test_sampcd_processor.py:439]: 将这个单测加入到全量单测中的原因是？

**Review by lelelelelez — CHANGES_REQUESTED** [review]
单测既然写了，在paddle_build.sh中cpu_py2这条ci里加个逻辑，检测到有人修改这个文件了，就跑一下这个单测。否则这个单测会将成为"历史中的无效代码"。

File: tools/test_sampcd_processor.py (hunk 1 above)
Comment (wadefelix) [reply tools/test_sampcd_processor.py:439]: > 将这个单测加入到全量单测中的原因是？

参考回复。还是加到单元测试中吧，任何代码都应该保证质量，都应该有足够的测试保证。这是好的实践。

**Review by wadefelix — COMMENTED** [review]
> 单测既然写了，在paddle_build.sh中cpu_py2这条ci里加个逻辑，检测到有人修改这个文件了，就跑一下这个单测。否则这个单测会将成为"历史中的无效代码"。

https://github.com/PaddlePaddle/Paddle/pull/31863/commits/6d2de971ea08a29dfb91b3d65d9c7f3ffea03a5f
done, added in the `tools/check_file_diff_approvals.sh`

**Review by lelelelelez — APPROVED** [review]
LGTM

**Pull request merged**